package.json

```diff
@@ -81,14 +73,8 @@
   "lint-staged": {
     "packages/**/*.{js,jsx,ts,tsx}": [
       "pretty-quick --staged",
-      "npm run lint",
+      "pnpm run lint",
       "git add"
     ]
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged",
-      "pre-push": "jest"
-    }
   }
```

Comment:
I think yes
https://blog.typicode.com/husky-git-hooks-javascript-config/
This is explaining it

> Since v0, husky was configuring Git hooks using JavaScript, either with .huskyrc.js files or fields in package.json.
> 
> Now Git hooks are configured using individual files for hooks in a .husky/ directory.